frontend/include/chpl/types/CompositeType.h

```diff
@@ -206,6 +206,27 @@ class CompositeType : public Type {
 
     return types::QualifiedType();
   }
+
+  /** Get the string type */
+  static const RecordType* getStringType(Context* context);
+
+  /** Get the range type */
+  static const RecordType* getRangeType(Context* context);
+
+  /** Get the bytes type */
+  static const RecordType* getBytesType(Context* context);
+
+  /** When compiling without a standard library (for testing purposes),
+      the compiler code needs to work around the fact that there
+      is no definition available for the record types needed
+      by the language but provided in the library (such as 'string').
+      This function allows code to easily detect that case.
+   */
+  static bool isMissingBundledType(Context* context, ID id);
```

Comment:
BTW, since this is no longer "missing bundled __record__ type", are we sure the case below that sets a return type to `asttags::Record` correct? Would this function return true for non-record missing types, too?

Reply:
Perhaps we want to split this into two additional helper functions, `isMissingBundledRecordType` and `isMissingBundledClassType`, and have `isMissingBundledType` call them both?